packages/core/src/executors/init/platform/plugins.ts

```diff
@@ -11,10 +11,7 @@ export const execute = (options: InitOptions, config: Config) => {
     platform: "ios" | "android"
   ) => {
     for (const installedPlugin of installedPlugins) {
-      const bundle = require(path.project.resolve(
-        "node_modules",
-        installedPlugin
-      ));
+      const bundle = require(path.hoist.resolve(installedPlugin));
```

Comment:
NIT: this was originally an error because of the nohoist policy - without the nohoist policy with respect to all packages i.e. `**` now all the symlinked monorepo packages are on the same level. Previously this would fail the local plugin. The `path.hoist.resolve` solved this because it would dynamically look up the tree from the current ctx.

After thinking about the `path.hoist.resolve` we may want to have a safe resolve on that because if they don't expose the `package.json` via the `exports` if they are using `exports` it will fail packages.